package/src/components/ChannelPreview/hooks/useLatestMessagePreview.ts

```diff
@@ -286,19 +286,6 @@ export const useLatestMessagePreview = <
     : '';
 
   const [readEvents, setReadEvents] = useState(true);
-  const [latestMessagePreview, setLatestMessagePreview] = useState<
-    LatestMessagePreview<StreamChatGenerics>
-  >({
-    created_at: '',
```

Comment:
This change should be for the better (since the default state seems very broken anyway), but please check and make sure that we don't have code in the downstream components that depends on this (since it's being removed).

Reply:
Yeah, I don't see any noticeable bug post this change